idl/github.com/uber/tchannel/meta.thrift

```diff
@@ -22,7 +39,7 @@ struct VersionInfo {
 }
 
 service Meta {
-    HealthStatus health()
+    HealthStatus health(1: HealthRequest hr)
```

Comment:
Is adding this argument backwards compatible? I had thought that new params were breaking...